bin/BuildPackages.sh

```diff
@@ -236,17 +209,13 @@ build_one_package() {
       build_carat
     ;;
 
-    cohomolo*)
```

Comment:
That code was kept from the old `BuildPackages.sh` unchanged. I'm happy to believe it was bad, I should have checked when I ported it across. i don't use the package.